reconcile/test/test_utils_state.py

```diff
@@ -0,0 +1,65 @@
+import pytest
+import boto3
+from moto import mock_s3
+from reconcile.utils.state import State
+
+
+@pytest.fixture
+def accounts():
+    """Account name is needed to instantiate a State"""
+    return [{'name': 'some-account'}]
+
+
+@pytest.fixture
+def s3_client(monkeypatch):
+    monkeypatch.setenv('AWS_ACCESS_KEY_ID', 'testing')
+    monkeypatch.setenv('AWS_SECRET_ACCESS_KEY', 'testing')
+    monkeypatch.setenv('AWS_SECURITY_TOKEN', 'testing')
+    monkeypatch.setenv('AWS_SESSION_TOKEN', 'testing')
+    monkeypatch.setenv('APP_INTERFACE_STATE_BUCKET', 'some-bucket')
+    monkeypatch.setenv('APP_INTERFACE_STATE_BUCKET_ACCOUNT', 'some-account')
+
+    with mock_s3():
+        s3_client = boto3.client('s3', region_name='us-east-1')
+        yield s3_client
+
+
+def test_ls_returns_correct_file(accounts, s3_client, mocker):
+    s3_client.create_bucket(Bucket='some-bucket')
+    s3_client.put_object(Bucket='some-bucket',
+                         Key='state/integration-name/some-file-1',
+                         Body='test')
+
+    """Creating some-file-2 to identify when two or more integrations have
```

Comment:
nit: `"""` is generally only used for docstrings as far as I'm aware.  I wouldn't make a habit of doing this and would just use `#` instead.  If linters aren't catching this now, they might in the future.

Reply:
Done.